src/lib/algolia-outer-wrapper.js

```diff
@@ -38,7 +36,11 @@ export default function algoliaOuterWrapper (NextWrapper, Page) {
         searchState = Page.buildAlgoliaStates(appContext)
       }
       // We should always configure this in the state to prevent a onSearchStateChange first render
-      searchState.configure = { ...searchState.configure, hitsPerPage: Config.get().algoliaResultsPerPage, distinct: MAX_VARIANTS_PER_PRODUCT }
+      searchState.configure = {
+        ...searchState.configure,
+        hitsPerPage: Config.get().algoliaResultsPerPage,
+        distinct: Config.get().algoliaDistinctVariants || true
```

Comment:
In the PR you say:
> When this is not set we should expect the default value of false

But this seems to be defaulting to `true`?

Reply:
Sorry, it should default to true :+1: